### ISSUE TITLE
Incorrect target listing for sys.db classes

### ISSUE THREAD

**Issue by elnabo** [HaxeFoundation/haxe#6540]
Classes such as [sys.db.Mysql](http://api.haxe.org/sys/db/Mysql.html) and [sys.db.Sqlite](http://api.haxe.org/sys/db/Sqlite.html) are said to target Python in the documentation.

However, there is no code for this target, resulting in the default execution error : `Not implemented for this platform`

**Comment by ibilon** [comment]
This could be fixed by adding a meta such as `@:requireTargetImpl` to the classes like this in `std/` and if there's no file in `std/$target/_std/` matching then not adding it in the list of supported target.

Could be very useful outside of dox too, if that's the case haxe could give a compile time error about it, instead of the runtime exception.

May even be useful outside of the std with code which requires files like `Class.js.hx`.

**Comment by Simn** [comment]
I wonder why these classes aren't externs...

**Comment by ibilon** [comment]
If they were extern they'd still pass haxe compilation and fail at runtime though.

**Comment by Simn** [comment]
I know, but still!

**Comment by Simn** [comment]
Agreed on adding some metadata for this, but all we have to do is check if that metadata exists and ignore the type for documentation generation. That works because the core-type is only going to surface if there is no target-specific implementation.
